### PR TITLE
Try to fix Circle & clean up a little

### LINKED ISSUE #214: Circle is broken

### PR DESCRIPTION
**Pull request by hoechenberger**
Fixes #214

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -25,24 +25,28 @@ jobs:
...
             name: Install dependencies in conda base environment
+            shell: bash -l
             command: |
+              conda activate base
               conda update -n base -c defaults conda
-              pip install numpy scipy pandas json_tricks matplotlib seaborn nibabel coloredlogs python-picard fire
+              echo `which python`
+              echo `which pip`
+              pip install numpy scipy scikit-learn pandas json_tricks matplotlib \
+                          seaborn nibabel coloredlogs python-picard fire
```
Comment (agramfort) [inline .circleci/config.yml:45]: we should put this in a requirements.txt file

**Comment by hoechenberger** [comment]
LOL no waaaayyyy!! All the time it was just pip… aargh

File: .circleci/config.yml (hunk 1 above)
Comment (hoechenberger) [reply .circleci/config.yml]: yes I guess so

**Comment by agramfort** [comment]
just a cache issue https://app.circleci.com/pipelines/github/mne-tools/mne-study-template/913/workflows/34ba2a0a-02f3-45ba-8913-7750ea98891d/jobs/1113

feel free to MRG if you want

**Pull request merged**